src/hotspot/share/oops/compressedOops.inline.hpp

```diff
@@ -25,7 +25,6 @@
 #ifndef SHARE_OOPS_COMPRESSEDOOPS_INLINE_HPP
 #define SHARE_OOPS_COMPRESSEDOOPS_INLINE_HPP
 
-#include "gc/shared/collectedHeap.hpp"
```

Comment:
`utilities/globalDefinitions.hpp` for `*PTR_FORMAT` and others is missing.

Reply:
Fixed.